src/components/TitleBarButtons.vue

```diff
@@ -6,6 +6,15 @@
     text-color="secondary"
     class="full-height cursor-not-allowed no-border-radius"
   >
+    <q-btn
+      dense
+      flat
+      round
+      icon="push_pin"
+      color="black"
+      class="title-bar-buttons"
+      @click="changePinWindow()"
+    ></q-btn>

```

Comment:
Maybe the icon is not changing depending on the status of `isPinned`?

Reply:
No, I don't think so, I placed some break points and found that electron just doesn't trigger `on("always-on-top-changed")` when it's set by another program, while other parts work flawlessly. This also happens on other electron apps like [clash for windows](https://github.com/Fndroid/clash_for_windows_pkg). As a result, there seems to be no way to fix it without changing electron itself.
On my view it's not a serious problem since users won't have to use another software to pin while it can be done in VOICEVOX and the bug only affects the icon, the state of window changes whenever it's set by another program or itself.

Reply:
Oops, I think we may have a misunderstanding.
What I meant to say is that the icon may not change when `v-if="$q.platform.is.mac"`.

I understand that the appearance of the icon does not change when external software pins it.
As you said, I don't think this is a serious problem.

Reply:
Yes, that makes sense, I've fixed it in the last commit. The weird code comes when I realized that the `v-if="$q.platform.is.mac"` will only be true on mac and decided to work on the Windows part first, then I forgot its existence. Also I don't have a mac so the performance there can't be guaranteed.